crates/bevy_sprite/src/texture_atlas.rs

```diff
@@ -19,18 +19,24 @@ pub struct TextureAtlas {
     pub size: Vec2,
     /// The specific areas of the atlas where each texture can be found
     pub textures: Vec<Rect>,
-    pub texture_handles: Option<HashMap<Handle<Image>, usize>>,
+    /// Mapping from texture handle to index
+    pub(crate) texture_handles: Option<HashMap<Handle<Image>, usize>>,
 }
 
 #[derive(Component, Debug, Clone, Reflect)]
 pub struct TextureAtlasSprite {
+    /// The tint color used to draw the sprite, defaulting to [`Color::WHITE`]
     pub color: Color,
+    /// Texture index in [`TextureAtlas`]
     pub index: usize,
+    /// Whether flip the sprite in x axis
     pub flip_x: bool,
+    /// Whether flip the sprite in y axis
     pub flip_y: bool,
     /// An optional custom size for the sprite that will be used when rendering, instead of the size
```

Comment:
how about 
```rust
/// An optional custom size for the sprite that will be resized to when rendering
pub custom_size: Option<Vec2>,
```

Reply:
I think probably the original description:
```
/// An optional custom size for the sprite that will be used when rendering, instead of the size
/// of the sprite's image in the atlas
```
was fine and doesn't need to be changed.

Reply:
yea, will revert back